doc/getting_started.md

```diff
@@ -0,0 +1,83 @@
+# Getting started
+
+This document will guide through setting up PyDotBot connected to a DotBot
+gateway and a DotBot robot.
+
+## Prerequisites
+
+Make sure you have access to an nRF DK board (nrf52833dk, nrf52840dk or
+nrf5340dk) and to a DotBot (v1 or v2).
+
+Follow the instructions of the
+[DotBot firmware getting started page][dotbot-firmware-getting-started].
+
+## Install PyDotBot
+
+Use pip to install the latest version of PyDotBot from [pypi][pydotbot-pypi]:
+
+```
+pip install pydotbot -U
+```
+
+## Setup the gateway
+
+The gateway is used to bridge the UART communication between PyDotBot running on
+a computer and the BLE radio used to communicate wirelessly with the DotBot(s).
+
+1. Connect the nRF DK gateway to your computer
+
+2. Identify the TTY port it is connected to. On Linux, it should be `/dev/ttyACM0`.
+  On Windows, check the device manager, it should be `COM1`, `COM2`, `COM3`, etc
```

Comment:
maybe a technical note, but sometimes DKs show up as two difference COM ports?

Reply:
yes the nrf53dk shows 2 COM ports, I don't which one is the right one on windows. On Linux it's the lowest value among `/dev/ttyACMx` and `/dev/ttyACMy`